src/tests/ftest/util/server_utils_params.py

```diff
@@ -440,7 +440,7 @@ def __init__(self, index=None, provider=None):
             #       AIO /tmp/aiofile AIO1 4096
             self.scm_mount = BasicParameter(None, "/mnt/daos")
             self.scm_class = BasicParameter(None, "ram")
-            self.scm_size = BasicParameter(None, 16)
+            self.scm_size = BasicParameter(None, 4)
```

Comment:
This is for more than just VM. Is there potential to break HW tests?

Reply:
Yes, moved this change to the yaml files.